src/components/PastTripCard.js

```diff
@@ -0,0 +1,21 @@
+import pastTripList from '../data';
+import styled from 'styled-components';
+import PastTripList from './PastTripList';
+
+export default function PastTripCard() {
+  return (
+    <PastTripCards role="list" aria-label="past-trips">
+      {pastTripList.map(({ country, city, _id }) => {
+        return <PastTripList country={country} city={city} key={_id} />;
```

Comment:
Maybe you can try to delete the return Statement :)

Reply:
Thank you Sven :) The return statement is necessary at this point otherwise I get the following warning: 'Array.prototype.map() expects a return value from arrow function'.

Reply:
My pleasure :) Maybe you can try the following code to see if it works without an error message and return statement if you want. 

```js
<PastTripCards role="list" aria-label="past-trips">
      {pastTripList.map(({ country, city, _id }) => (
       <PastTripList country={country} city={city} key={_id} />;
      ))}
```